### PR TITLE
Updated WebSphere Application Server with 2nd fix PH42762

### PR DESCRIPTION
**Pull request by hubje2020**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [ ] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [ ] Status: please select a value from the status table at the top
  - [ ] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [ ] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [ ] Please mind the sorting

### CODE REVIEW

**Comment by grayum** [comment]
Thanks for the PR. However due to several conflicts could you please fetch, rebase and fix the branch before you send an update PR. This is probably due to an old local repo.

**Comment by grayum** [comment]
We've added new columns in regards to added CVE's. Please fetch, update accordingly and issue a new PR. Thank you.

File: software/README.md [hunk 1]
```diff
@@ -1085,8 +1086,8 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | IBM | Sterling Inventory Visibility | Unknown | Vulnerable | | [source](https://www.ibm.com/support/pages/node/6525544)|
 | IBM | Sterling Order Management | Unknown | Not vuln | | [source](https://www.ibm.com/support/pages/node/6525544)|
 | IBM | VM Manager Tool (part of License Metric Tool) | >9.2.21,<9.2.26 | Vulnerable | | [source](https://www.ibm.com/support/pages/node/6525762/)|
-| IBM | Websphere | 8.5 | Vulnerable | fix: PH42728 | [source](https://www.ibm.com/support/pages/node/6525706/)|
-| IBM | Websphere | 9.0 | Vulnerable | fix: PH42728 | [source](https://www.ibm.com/support/pages/node/6525706/)|
+| IBM | WebSphere | 8.5 | Vulnerable | fix: PH42728, PH42762 | [source](https://www.ibm.com/support/pages/node/6525706/), [source](https://www.ibm.com/support/pages/node/6526750)|
```
Comment (maertsen) [inline software/README.md:1089]:
```suggestion
| IBM | WebSphere | 8.5 | | Workaround | Workaround | | apply hotfixes PH42728, PH42762 | [source](https://www.ibm.com/support/pages/node/6525706/), [source](https://www.ibm.com/support/pages/node/6526750)|
```

File: software/README.md [hunk 2]
```diff
@@ -1085,8 +1086,8 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
...
+| IBM | WebSphere | 9.0 | Vulnerable | fix: PH42728, PH42762 | [source](https://www.ibm.com/support/pages/node/6525706/), [source](https://www.ibm.com/support/pages/node/6526750)|
```
Comment (maertsen) [inline software/README.md:1090]:
```suggestion
| IBM | WebSphere | 9.0 | | Workaround | Workaround | | apply hotfixes PH42728, PH42762 | [source](https://www.ibm.com/support/pages/node/6525706/), [source](https://www.ibm.com/support/pages/node/6526750)|
```

File: software/README.md [hunk 3]
```diff
@@ -1072,6 +1072,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Supplier        | Product         | Version (see Status) | Status          | Notes           | Links |
 |:----------------|:----------------|:--------------------:|:---------------:|:----------------|-----------:|
 | IBM | All products | | Investigation | | [source](https://www.ibm.com/blogs/psirt/an-update-on-the-apache-log4j-cve-2021-44228-vulnerability/)|
+| IBM | Business Automation Workflow | 18.0.0+ | Fix | JR64456 / JR64096 | [source](https://www.ibm.com/support/pages/node/6525834)|
```
Comment (maertsen) [inline software/README.md:1075]:
```suggestion
| IBM | Business Automation Workflow | 18.0.0+ |  | Workaround |  |  | apply JR64456 / JR64096 | [source](https://www.ibm.com/support/pages/node/6525834)|
```

**Review by maertsen — CHANGES_REQUESTED** [review]
These are the minimal changes required to merge @hubje2020. Any updates from your side are appreciated.

**Comment by grayum** [comment]
No updated PR in 5 days. Closing this one.

**Pull request closed without merging**